tutorials/preprocessing/plot_35_artifact_correction_regression.py

```diff
@@ -0,0 +1,173 @@
+# -*- coding: utf-8 -*-
+"""
+.. _tut-artifact-regression:
+
+Repairing artifacts with regression
+===================================
+
+This tutorial covers removal of artifacts using regression as in
+:footcite:`GrattonEtAl1983`.
+
+.. contents:: Page contents
+   :local:
+   :depth: 2
+
+We begin as always by importing the necessary Python modules and loading some
+data. In this case we use data from :ref:`brainstorm <tut-brainstorm-auditory>`
+because it has both EOG and ECG electrodes. We then crop it to 60 seconds,
+set some channel information, then process the data.
+
+Note that there are other corrections that are useful for this dataset that we
+will not apply here — see :ref:`tut-brainstorm-auditory` for more information.
+"""
+
+import os.path as op
+import numpy as np
+import mne
+
+data_path = mne.datasets.brainstorm.bst_auditory.data_path()
+raw_fname = op.join(data_path, 'MEG', 'bst_auditory', 'S01_AEF_20131218_01.ds')
+raw = mne.io.read_raw_ctf(raw_fname).crop(0, 60)
+raw.set_channel_types({'HEOG': 'eog', 'VEOG': 'eog', 'ECG': 'ecg'})
+raw.pick_types(meg=True, stim=True, misc=True,
+               eog=True, ecg=True, ref_meg=False).load_data()
+raw.info['bads'] = ['MLO52-4408', 'MRT51-4408', 'MLO42-4408', 'MLO43-4408']
+raw.filter(None, 40)
+decim = 12  # 2400 -> 200 Hz sample rate for epochs
+
+###############################################################################
+# Removing artifacts by regression
+# ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+#
+# Generally speaking, artifacts whose time waveforms on the sensors are
```

Comment:
To me using "whose" implies that we speak about a person. Of course I'm not a native speaker, but would it be better to change this to "artifact waveforms that are accurately reflected by some reference signal"?